nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/QueryElasticsearchHttp.java

```diff
@@ -267,30 +267,28 @@ public void setup(ProcessContext context) {
 
     @Override
     public void onPropertyModified(final PropertyDescriptor descriptor, final String oldValue, final String newValue) {
+        final Set<Relationship> routeQueryInfoRels = new HashSet<>();
+        routeQueryInfoRels.add(REL_SUCCESS);
+        routeQueryInfoRels.add(REL_FAILURE);
+        routeQueryInfoRels.add(REL_RETRY);
+
+        final Set<Relationship> successRels = new HashSet<>();
```

Comment:
I don't think even needs to exist at this point because `routequeryInfoRels` effectively does all three use cases and only gets the 4th relationship based on the routing strategy. Might want to refactor that and change the name to be more appropriate to the overlap.

Reply:
I thought the same thing.  Thanks for the feedback my latest did take that approach